frontend/packages/console-shared/src/components/error/fallbacks/ErrorDetails.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
```

Comment:
I would suggest to put this component into `frontend/packages/console-shared/src/components/modals` and rename into `ErrorDetailsModal`